### PR TITLE
HARVESTER: support custom log & productName

### PR DESCRIPTION
**Pull request by WuJun2016**
### Summary
1. support customization of Harvester Web Page Name / Tab Name  (single mode / mcm mode)
2. fix mcm issue bug

#### PR Checklist
~- Is this a multi-tenancy feature/bug?
    - [ ] Yes, the relevant RBAC changes are at:~
~- Do we need to backport changes to the [old Rancher UI](https://github.com/rancher/u), such as RKE1?
    - [ ] Yes, the relevant PR is at:~
~- Are backend engineers aware of UI changes?
    - [ ] Yes, the backend owner is:~

Fixes #

- https://github.com/harvester/harvester/issues/3396

### CODE REVIEW

File: pkg/harvester/l10n/zh-hans.yaml [hunk 1]
```diff
@@ -1028,6 +1028,7 @@ advancedSettings:
     'harv-auto-disk-provision-paths': 指定 Harvester 将自动添加为虚拟机存储的磁盘（使用 glob 模式）。
     'harv-support-bundle-image': Support Bundle 镜像配置。前往 <a href="https://hub.docker.com/r/rancher/support-bundle-kit/tags" target="_blank">rancher/support-bundle-kit</a> 查看更多版本。
     'harv-release-download-url': 此设置用于配置<code>升级版本下载</code>的 URL 地址。Harvester 将从该 URL 托管的 (<code>$URL</code>/<code>$VERSION</code>/version.yaml) 文件中获取 ISO URL 和校验和值。
+    'harv-rancher-manager-support': Multi-cluster provisioning and management of Kubernetes clusters.
```
Comment (n313893254) [inline pkg/harvester/l10n/zh-hans.yaml:1031]: Should be Chinese

File: pkg/harvester/models/management.cattle.io.setting.js [hunk 2]
```diff
@@ -27,6 +33,24 @@ export default class HciSetting extends HarvesterResource {
     return null;
   }
 
+  goToEdit(moreQuery = {}) {
+    const location = this.detailLocation;
+
+    if (location?.params?.id === 'ui-pl') {
```
Comment (n313893254) [inline pkg/harvester/models/management.cattle.io.setting.js:39]: `ui-pl` should import from `pkg/harvester/config/settings.js`

**Review by n313893254 — CHANGES_REQUESTED** [review]
Others LGTM

File: pkg/harvester/models/management.cattle.io.setting.js (hunk 2 above)
Comment (WuJun2016) [reply pkg/harvester/models/management.cattle.io.setting.js]: Done

**Comment by WuJun2016** [comment]
PTAL again, thanks

**Review by n313893254 — APPROVED** [review]

**Pull request merged**